apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ExperimentsResourceTest.java

```diff
@@ -1626,6 +1627,87 @@ void getById() {
         }
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    class DeleteExperimentItems {
+
+        @Test
+        void deleteExperimentsById__whenExperimentExists__thenReturnNoContent() {
+            var experiment = podamFactory.manufacturePojo(Experiment.class);
+
+            createAndAssert(experiment, API_KEY, TEST_WORKSPACE);
+
+            deleteExperimentAndAssert(Set.of(experiment.id()), API_KEY, TEST_WORKSPACE);
```

Comment:
Minor: after the creation call needs a retrieval to check the experiment exits, as this only asserts that the creation call succeeded. 

It will make the test much more endurable.

This comment applies to all tests.